### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by tanja84dk** [xthursdayx/docker-gpodder#8]
Sorry to ask because I'm not sure how it works on github prober
How is the correct way for me to send you a patch / possible improvement of the container?

The reason is that I'm in the middle of testing several changes ( Working on a local clone of my fork to make sure not to impact your files ) to lower the image size and still working as normal.
The reason is that I noticed that some oh the things in the container is done in a way I wouldn't say is the best way and also something there are just raising the size without actually doing anything for the image ( of what I see )

**Comment by xthursdayx** [comment]
Hi @tanja84dk, thanks for your interest in working on the image! I'm self-taught and have learned as I go; figuring out how to fix problems in my own projects as they arise - there are definitely holes in my knowledge and I'm sure there are better ways to do things, so I'm open to help and suggestions!

My advice is for you to get your fork working as you'd like and then submit a PR (pull request) to my primary repository. So long as there are not any major conflicts, I'll probably just test your changes and then merge our branches and push the image to Docker Hub. If the changes will "break" the image as it currently exists, then it might require more work on my part, but I'm happy to work with you on it, so feel free to submit your PR once you're ready. Cheers!

**Comment by tanja84dk** [comment]
I will do that and thanks a lot.
Btw I'm currently working on my Dev branch ( Tanja84DK-Dev-3.10.19 ) and right now if you want a look.
And the main change I have done right now is using the tar src file instead of git clone both because clone gives a lot of overhead that the container doesn't need and also I would say its more stable because using release src instead of clone also makes sure that  the container isn't tested.

And I'm also testing using the release instead of libgtk-3 dev package because I cant see the extra overhead is needed.
And right now I got it down to 1.1 GB instead of 1.5 GB and going to further test the next week to see if something makes issues

**Comment by xthursdayx** [comment]
I can't find your other comment regarding you using your fork vs my updated image, but as I mentioned previously, if you develop your fork and feel like you find better/more efficient solutions for running things I'm happy for you to contribute a PR. Of course, you're under no obligation to contribution, and you're also welcome to publish your own image if you'd prefer. I just wanted to be clear that I'm fully in support of collaboration vs competition and don't have any sense of this container being propriety or "mine". I just care about working with folks to develop code that works as well as it possibly can!

**Comment by tanja84dk** [comment]
I have dropped it and also closed the ticked because the the way I would say is best would mess up your build runner build chains because I would have used the tar release to not add the overhead of git and the git history overhead. And you have also got the image smaller then mine were in the last update.

And I'm so sorry that I got so stressed with c++ study and work so forgot to add the update in here when I closed it.
But I do look over all your merge's runner tests etc because my phohe is still following all updates so I do will try to add a comment to you if I spot something.

I'm so sorry.
